modules/nlu/src/backend/dialog-conditions/misunderstood.ts

```diff
@@ -22,6 +30,7 @@ export default {
       .get('confidence', 0)
       .value()
 
-    return Math.max(highest_none, oos)
+    const max = Math.max(highest_none, oos)
+    return params.maxConfidence ? max * (params.maxConfidence / 100) : max
```

Comment:
I'm sorry I don't understand why someone would change the confidence of the input being out of scope ? And surely why do a multiplication? Wha'ts the usecase of downranking the our of scope confidence

Reply:
Talk with Sylvain for that one, he asked me to make that change. Basically it's to prevent misunderstood to fire at 100% confidence, when you might definitely prefer another trigger which fires at 70 or 80% instead